support/index.html

```diff
@@ -81,6 +81,16 @@ <h4>OME Training</h4>
                     </div>
                 </div>
             </div>
+            <div class="small-12 medium-4 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>On the web</h4>
+                        <p class="card-caption">Follow us on social media or listen in to a community meeting.</p>
```

Comment:
did not find the link to the zoom, maybe it is either ``read the minutes`` instead, or maybe this is WiP ?

Reply:
Yes, this goes back to https://github.com/ome/www.openmicroscopy.org/issues/402 -- I assume we need to publish it but we should all agree. That _likely_ means activating the waiting room.

Reply:
Could ask in AOB today.